applications/stream-applications-core/stream-applications-security-common/src/test/java/org/springframework/cloud/stream/app/security/common/SecurityEnabledAuthorizedAccessTests.java

```diff
@@ -73,6 +73,7 @@ public void testHealthEndpoint() {
 	@Test
 	@SuppressWarnings("rawtypes")
 	public void testInfoEndpoint() {
+		restTemplate.getRestTemplate().getInterceptors().clear();
```

Comment:
So this test is really doing a "when security is enabled and I am logged in I can access X,Y,Z" so I think we should not clear out the interceptors.

Reply:
This really brings up a good point (which is outside of the scope of this PR) that a 1-liner high level description  on each of these tests would go a long way as they are all very similiar looking and take a minute to get my head around which one is doing what.